Mastonet/TimelineWebSocketStreaming.cs

```diff
@@ -115,3 +121,8 @@ public override void Stop()
         base.Stop();
     }
 }
+
+[JsonSerializable(typeof(TimelineMessage))]
+internal partial class TimelineMessageContext : JsonSerializerContext
```

Comment:
Can we put this class at the same level of `TimelineMessage`, so we can keep the visibility private ?

Reply:
I don't think so. If I add it right below TimelineMessage it throws three errors, and "fixing" (i.e. let Visual Studio fix) those throws an error in the Deserialize call that uses the context.